webpack.config.js

```diff
@@ -1,27 +1,21 @@
 const path = require('path');
 const webpackConfig = require('terra-dev-site/src/config/webpack.config');
-const I18nAggregatorPlugin = require('./packages/terra-i18n-plugin');
-const i18nSupportedLocales = require('./packages/terra-i18n/lib/i18nSupportedLocales');
+const aggregateTranslations = require('./packages/terra-i18n/scripts/aggregate-translations/aggregate-translations');
+
+aggregateTranslations({ baseDirectory: __dirname });
 
 const processPath = process.cwd();
 const rootPath = processPath.includes('packages') ? processPath.split('packages')[0] : processPath;
-const translationsPath = path.join(rootPath, 'packages', 'terra-site');
-
-const i18nPlugin = new I18nAggregatorPlugin({
-  baseDirectory: path.resolve(translationsPath),
-  supportedLocales: i18nSupportedLocales,
-});
-
-const plugins = webpackConfig.plugins;
-plugins[2] = i18nPlugin;
 
-webpackConfig.plugins = plugins;
-
-webpackConfig.resolve.modules = [path.resolve(translationsPath, 'aggregated-translations'), 'node_modules'];
+// Remove the terra-i18n-plugin from the configuration
```

Comment:
I was trying to figure out why you were doing this. The plugin is added in terra-dev-site.
These mono-repos sure lead to some fun chicken and egg problems. :)